core-rust/state-manager/src/store/in_memory.rs

```diff
@@ -123,6 +125,7 @@ impl InMemoryStore {
             epoch_proofs: BTreeMap::new(),
             vertex_store: None,
             substate_store: InMemorySubstateDatabase::standard(),
+            node_metadatas: BTreeMap::new(),
```

Comment:
As discussed on a call, this gets renamed to `node_ancestry_record`s, to avoid 2 issues (the plural naming and the non-existence of records for roots)